seacrowd/sea_datasets/iapp_squad/iapp_squad.py

```diff
@@ -0,0 +1,141 @@
+# coding=utf-8
+import json
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_DATASETNAME = "iapp_squad"
+_CITATION = """\
+            @dataset
+            {
+              kobkrit_viriyayudhakorn_2021_4539916,
+              author       = {Kobkrit Viriyayudhakorn and Charin Polpanumas},
+              title        = {iapp_wiki_qa_squad},
+              month        = feb,
+              year         = 2021,
+              publisher    = {Zenodo},
+              version      = 1,
+              doi          = {10.5281/zenodo.4539916},
+              url          = {https://doi.org/10.5281/zenodo.4539916}
+            }
+            """
+
+_DESCRIPTION = """
+            `iapp_wiki_qa_squad` is an extractive question answering dataset from Thai Wikipedia articles.
+            It is adapted from [the original iapp-wiki-qa-dataset](https://github.com/iapp-technology/iapp-wiki-qa-dataset)
+            to [SQuAD](https://rajpurkar.github.io/SQuAD-explorer/) format, resulting in
+            5761/742/739 questions from 1529/191/192 articles.
+            """
+
+_HOMEPAGE = "https://github.com/iapp-technology/iapp-wiki-qa-dataset"
+_LICENSE = Licenses.MIT
+_HF_URL = " https://huggingface.co/datasets/iapp_wiki_qa_squad"
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "0.0.0"
+
+_IAPP = {
+    "train": "https://raw.githubusercontent.com/iapp-technology/iapp-wiki-qa-dataset/main/squad_format/data/train.jsonl",
+    "validation": "https://raw.githubusercontent.com/iapp-technology/iapp-wiki-qa-dataset/main/squad_format/data/valid.jsonl",
+    "test": "https://raw.githubusercontent.com/iapp-technology/iapp-wiki-qa-dataset/main/squad_format/data/test.jsonl",
+}
+
+
+class IappSquadDataset(datasets.GeneratorBasedBuilder):
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(name="iapp_squad_source", version=datasets.Version(_SOURCE_VERSION), description=_DESCRIPTION, subset_id="iapp_squad", schema="source"),
+        SEACrowdConfig(name="iapp_squad_seacrowd_qa", version=datasets.Version(_SOURCE_VERSION), description=_DESCRIPTION, subset_id="iapp_squad", schema="seacrowd"),
```

Comment:
I also recommend making `iapp_squad` a variable `_DATASETNAME` and setting it to the config `name` and `subset_id`